openml/runs/functions.py

```diff
@@ -111,6 +135,44 @@ def _run_exists(task_id, setup_id):
         assert(exception.code == 512)
         return False
 
+def _get_seeded_model(model, seed=None):
+    '''Sets all the non-seeded components of a model with a seed.
```

Comment:
But it can raise an exception:

```
import numpy as np
import sklearn.ensemble

import openml


rf = sklearn.ensemble.RandomForestClassifier(
    random_state=np.random.RandomState(1))
openml.runs.functions._get_seeded_model(rf, 5)
```
But you're right, it should be documented in the `run_tasks()` function.